### PR TITLE
Fix some go-lint warning

### PR DESCRIPTION
**Pull request by Hellcatlk**
Signed-off-by: ZouYu <zouy.fnst@cn.fujitsu.com>

### CODE REVIEW

File: upup/pkg/fi/cloudup/dns.go [hunk 1]
```diff
@@ -35,11 +35,11 @@ import (
 )
 
 const (
-	// This IP is from TEST-NET-3
+	// PlaceholderIP is from TEST-NET-3
 	// https://en.wikipedia.org/wiki/Reserved_IP_addresses
 	PlaceholderIP  = "203.0.113.123"
 	PlaceholderTTL = 10
-	// DigitalOcean's DNS servers require a certain minimum TTL (it's 30), keeping 60 here.
+	// PlaceholderTTLDigitialOcean digitalOcean's DNS servers require a certain minimum TTL (it's 30), keeping 60 here.
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/dns.go:42]: These two don't correctly express the intent.

File: upup/pkg/fi/cloudup/template_functions.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ type TemplateFunctions struct {
 	tags           sets.String
 }
 
-// This will define the available functions we can use in our YAML models
+// AddTo will define the available functions we can use in our YAML models
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/template_functions.go:64]:
```suggestion
// AddTo defines the available functions we can use in our YAML models.
```

Comment (Hellcatlk) [reply upup/pkg/fi/cloudup/template_functions.go]: Thanks, done.

File: upup/pkg/fi/cloudup/dns.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ import (
 )
 
 const (
-	// This IP is from TEST-NET-3
+	// PlaceholderIP is from TEST-NET-3
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/dns.go:38]: I intended the earlier comment about comments not correctly expressing the intent to also apply here. It seems minor enough, so I won't block on it.

**Pull request merged**